Atomic/syscontainers.py

```diff
@@ -669,6 +669,17 @@ def _canonicalize_location(self, destination):
             sysroot.load()
             osname = sysroot.get_booted_deployment().get_osname()
             destination = os.path.realpath(os.path.join("/ostree/deploy/", osname, os.path.relpath(destination, "/")))
+            # Test that the ostree location and /var show the same file.
+            # If they do not then we must return /var.
+            test_file = str(uuid.uuid4())
+            with open(os.path.join(destination, '.' + test_file), 'w') as f:
+                f.write('')
+            if not os.path.isfile('/var/.' + test_file):
```

Comment:
same here, let's not assume `/var` but whatever 'destination' is pointing to

Reply:
If we reuse destination here we will end up checking the same path we wrote out as destination is provided as a parameter for the function, then overridden within the block to be the ostree location.